lib/foreman/renderer.rb

```diff
@@ -105,6 +105,18 @@ def indent(count)
       prefix + text.gsub(/\n/, "\n#{prefix}")
     end
 
+    def dns_lookup(name_or_ip)
+      resolver = Resolv::DNS.new
+      Timeout.timeout(Setting[:dns_conflict_timeout]) do
+        ipquery = IPAddr.new(name_or_ip) rescue false
```

Comment:
@lzap Can we expect anything but a [IPAddr::Error](http://ruby-doc.org/stdlib-2.0.0/libdoc/ipaddr/rdoc/IPAddr/Error.html) here ?